os/mounting-storage.md

```diff
@@ -35,11 +37,13 @@ coreos:
         Description=Formats the ephemeral drive
         After=dev-xvdb.device
         Requires=dev-xvdb.device
+        ConditionPathExists=!/var/lib/format-xvdb
```

Comment:
This is specifically noted as a problem in the original issue https://github.com/coreos/docs/issues/843